doc/lms.mysql

```diff
@@ -1,7 +1,6 @@
 # --------------------------------------------------------
 # $Id$
 # --------------------------------------------------------
-
```

Comment:
Po co dodatkowa zmiana, która utrudni potem przenoszenie zmian między gałęziami?

Reply:
Usunięta zmiana. Minąłem to jakoś przed wypuszczeniem PR.